README.md

```diff
@@ -51,14 +51,24 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
 
 1. If you are running Mac OS X, a pkg file (which looks similar to Anaconda3-5.3.0-MacOSX-x86\_64.pkg) will be downloaded. Double-click on the file to start the installation process.
 1. After the installation has finished, open up a bash prompt in your terminal window.
+1. If you have an ARM64 Mac (M1/M2) running Catalina (or later), additional prerequisites must be installed for ISIS to run in emulation:
+ - Install [XQuartz](https://www.xquartz.org/). (Tested with XQuartz 2.8.5 on MacOS Catalina)
+ - Install Rosetta2. From the terminal run: `/usr/sbin/softwareupdate --install-rosetta --agree-to-license`
 1. Next setup your Anaconda environment for ISIS. In the bash prompt, run the following commands:
 
     ```bash
+    
+    #Setup the new environment as an x86_64 environment
```

Comment:
Can we indicate that this command is specific to installation on mac?  This portion of the installation process was previously platform-agnostic (w.r.t. supported platforms, at least), and these commands may break a linux install.